kv/memberlist/tcp_transport.go

```diff
@@ -14,7 +14,7 @@ import (
 
 	"github.com/go-kit/log"
 	"github.com/go-kit/log/level"
-	"github.com/hashicorp/go-sockaddr"
+	sockaddr "github.com/hashicorp/go-sockaddr"
```

Comment:
[nit] This change can be rolled back.

Reply:
I think it is a little more readable to be explicit about package naming considering we make the call to `sockaddr` and not `go-sockaddr`.  Happy to roll it back if desired.